test/FsAutoComplete.Tests.Lsp/Tests.fs

```diff
@@ -994,9 +994,8 @@ let tests =
     uriTests
     foldingTests
     linterTests
-    // commented out because this will only work in a netcoreapp3.0 context, which CI doesn't have.
-    //scriptPreviewTests
-    //scriptEvictionTests
-    //tooltipTests
-    //formattingTests
+    scriptPreviewTests
+    scriptEvictionTests
+    tooltipTests
+    formattingTests
```

Comment:
re-enabling the formatting tests caught the fantomas exception immediately. turning it on will be a big help here.

Reply:
👍